doc/user/radial_and_axial_expansion.rst

```diff
@@ -60,27 +57,235 @@ length, :math:`L_0`,
     \end{aligned}
     :label: diffHotNonRef
 
-Using Equations :eq:`diffHotNonRef` and :eq:`nonRefExp`, we can obtain an
-expression for the change in length, :math:`L_h - L_c`, relative to the
-non-reference temperature,
+Using Equations :eq:`diffHotNonRef` and :eq:`nonRefExp`, we can obtain an expression for the change
+in length, :math:`L_h - L_c`, relative to the non-reference temperature,
 
 .. math::
     \frac{L_h - L_c}{L_c} &= \frac{L_h - L_c}{L_0} \frac{L_0}{L_c}\\
     &= \left( \frac{L_h}{L_0} - \frac{L_c}{L_0} \right) \left( 1 + \alpha(T_c)\left(T_c - T_0\right) \right)^{-1}.
     :label: expNewRelative
 
-Using Equations :eq:`hotExp` and :eq:`nonRefExp`, we can simplify Equation
-:eq:`expNewRelative` to find,
+Using Equations :eq:`hotExp` and :eq:`nonRefExp`, we can simplify Equation :eq:`expNewRelative` to find,
 
 .. math::
     \frac{L_h - L_c}{L_c} = \frac{\alpha(T_h) \left(T_h - T_0\right) - \alpha(T_c)\left(T_c - T_0\right)}{1 + \alpha(T_c)\left(T_c - T_0\right)}.
     :label: linearExpansionFactor
 
 Equation :eq:`linearExpansionFactor` is the expression used by ARMI in
-:py:meth:`linearExpansionFactor
-<armi.materials.material.Material.linearExpansionFactor>`.
+:py:meth:`linearExpansionFactor <armi.materials.material.Material.linearExpansionFactor>`.
 
 .. note::
     :py:meth:`linearExpansionPercent
     <armi.materials.material.Material.linearExpansionPercent>` returns
     :math:`\frac{L - L_0}{L_0}` in %.
+
+Given that thermal expansion (or contraction) of solid components must conserve mass throughout the
+system, the density of the component is adjusted as a function of temperature based on Equation
+:eq:`hot_density`, assuming isotropic thermal expansion.
+
+.. math::
+    \rho(T_h) = \frac{\rho(T_0)}{\left(1 + \frac{\Delta L}{L_0}\right)^3} = \frac{\rho(T_0)}{\left(1 + \alpha_m (T_h) (T_h - T_0)\right)^3}
+    :label: hot_density
```

Comment:
```suggestion
:eq:`hot_density_general`, assuming isotropic thermal expansion.

.. math::
    \rho(T_h) = \frac{\rho(T_0)}{\left(1 + \frac{\Delta L}{L_0}\right)^3} = \frac{\rho(T_0)}{\left(1 + \alpha_m (T_h) (T_h - T_0)\right)^3}
    :label: hot_density_general
```
otherwise it refs the wrong equation. my bad on the poor suggested change! 
![image](https://github.com/user-attachments/assets/15ec50ec-67ab-4e78-b476-23c99f61e3a9)

Reply:
OH, nice catch man, thanks.

I guess I was too focused on the latex.